docs/source/tooling/remix.rst

```diff
@@ -0,0 +1,64 @@
+.. _tools.remix:
+
+Remix
+=====
+
+MythX is available as a plugin for `Remix <https://remix.ethereum.org>`_, a popular web-based IDE for smart contract development and deployment, created and hosted by the Ethereum Foundation.
+
+.. image:: img/remix.jpg
+
+Setup
+-----
+
+.. note:: These instructions will show the Remix interface that is current as of mid-2019. We recommend everyone use this interface.
+
+Because Remix is a web-based interface, no local installation in required. However, MythX will need to be activated specifically before use.
+
+To activate MythX in Remix:
+
+#. Click the :guilabel:`Plugins` icon (which resembles a plug).
+
+   .. figure:: img/pluginsbutton.png
+
+   Alternately, click the :guilabel:`See all Plugins` button under :guilabel:`Featured Plugins`.
+
+#. The full list of plugins for Remix will be displayed. Scroll down to the entry titled :guilabel:`MythX Security Verification` and click :guilabel:`Activate`.
+
+   .. figure:: img/mythxpluginlist.png
+
+   If done correctly, the plugin will be listed under :guilabel:`Active Modules` and the MythX logo will be shown in the sidebar.
+
+   .. figure:: img/activemodules.png
+
+#. *(Optional but recommended)* Click the MythX logo and enter your MythX credentials. This consists of your Ethereum address (also known as User ID) and the password supplied to you when you created your account. When done, click :guilabel:`Save`.
+
+   .. figure:: img/mythxcreds.png
+
+   .. note:: Without MythX credentials, you will be running in Trial Mode, which will only return a limited report of vulnerabilities. You can go to https://mythx.io to create a free account which will offer an unrestricted report.
+
+Usage
+-----
+
+You can perform a security analysis on any contract in any open file on Remix.
+
+To perform an analysis:
+
+#. Click the MythX logo on the sidebar to open the MythX control panel (if it isn't already open).
+
+#. Below the credentials section, there will be a select box containing a list of all applicable contracts. Select the one you wish to analyze and click the :guilabel:`Analyze` button.
+
+   .. figure:: img/analyze.png
+
+   .. note:: The contract may need to be compiled first, depending on the current Remix settings.
```

Comment:
I think we should specify how to compile the contract. Install `solidity compiler` from the list of plugin, select relevant compiler version and click on compile is the easiest way.

Reply:
I'll add some more details to the note about the Solidity plugin, unless you think it should be its own step, with images and everything.

Reply:
No I don't think we should be too specific since that is completely separated from MythX (we do not want people to think we do support work for the other plugin obviously).